src-tauri/src/lib.rs

```diff
@@ -21,11 +24,24 @@ impl AppState {
 #[cfg_attr(mobile, tauri::mobile_entry_point)]
 pub fn run() {
     Builder::default()
+        .plugin(tauri_plugin_shell::init())
+        .plugin(tauri_plugin_store::Builder::default().build())
         .setup(|app| {
-            app.manage(Mutex::new(AppState::default()));
+            let notes_store = app.handle().store_builder("notes.bin").build();
+
+            let local_note_service = LocalNoteService::new(notes_store);
+            let _remote_note_service =
+                RemoteNoteService::new("/notes".to_string(), HttpClientImpl::default());
```

Comment:
For future reference, I don't think we should be handling the remote side of things just yet, take it one at a time. The issue is called "offline save", and should only be dealing with such I would say?

Don't delete this anywyay, it's fine there. But we currently have an unused var in the code now, and if we decide about changing how we handle the system, it may never be deleted.